### ISSUE TITLE
Support SLurm

### ISSUE THREAD

**Issue by chufanchen** [TARTRL/TLaunch#1]
Slurm is a popular management system for Linux clusters. It would be nice if Tlaunch supported it. I would like to contribute and make that happen.

**Comment by huangshiyu13** [comment]
We appreciate contributions and new features for TLaunch. If you can implement a Slurm-version TLaunch, we'd like to merge it into our framework.

**Comment by chufanchen** [comment]
I tried to run `lp_ssh`, but `Tlaunch` import `get_config`, `rl_tables` etc from `TMARL` which I cannot find. Is it not open-source? 
Edit: Find it in `TiKick`.

**Comment by huangshiyu13** [comment]
Yes, this repo and TMARL have not been formally released. We will release our new RL framework recently. And we will polish the TLaunch repo after that.
